app/controllers/provider_suggestions_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ProviderSuggestionsController < ApplicationController
   def suggest
-    suggestions = ProviderSuggestion.suggest(params[:query])
+    sanitised_query = CGI.escape(params[:query])
```

Comment:
We are missing tests for this.

Could I suggest we follow the pattern we used in API to create a service that escapes the query param? We can then test the service and then for this route we mock the service.